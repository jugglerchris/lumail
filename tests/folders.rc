function dump_selected()
    io.write("Selected: offset="..maildir_offset().."\n")
    for i,f in ipairs(selected_folders()) do
       io.write(" "..i.."  "..f.."\n")
    end
end
set_selected_folder("output/md2")
dump_selected()
add_selected_folder("output/md1")
dump_selected()
add_selected_folder()
dump_selected()

set_selected_folder()
dump_selected()

set_selected_folder("output/md1")
dump_selected()
toggle_selected_folder("output/md2")
dump_selected()

toggle_selected_folder()
dump_selected()
