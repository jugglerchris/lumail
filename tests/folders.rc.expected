Selected: offset=0
 1  output/md2
Selected: offset=0
 1  output/md2
 2  output/md1
Selected: offset=0
 1  output/md2
 2  output/md1
Selected: offset=0
 1  output/md1
Selected: offset=0
 1  output/md1
Selected: offset=0
 1  output/md1
 2  output/md2
Selected: offset=0
 1  output/md2
Exit: 0
